types/parse/index.d.ts

```diff
@@ -1343,14 +1349,10 @@ declare global {
                 original?: T | undefined;
             }
 
-            interface AfterSaveRequest<T = Object> extends TriggerRequest<T> {
-                context: object;
-            }
+            interface AfterSaveRequest<T = Object> extends TriggerRequest<T>, ContextOption {}
```

Comment:
It looks like the context used to be required and is now optional.  Was that intentional?